h/static/scripts/group-forms/components/CreateEditGroupForm.tsx

```diff
@@ -254,12 +253,18 @@ export default function CreateEditGroupForm() {
           )}
           <div className="grow" />
           <SaveStateIcon
-            state={saving ? 'saving' : saved ? 'saved' : 'unsaved'}
+            state={
+              saveState === 'saving'
+                ? 'saving'
+                : saveState === 'saved'
+                  ? 'saved'
+                  : 'unsaved'
+            }
```

Comment:
I think `saveState === 'unmodified' ? 'unsaved' : saveState` should be equivalent, but please check 😅.

```suggestion
            state={
              saveState === 'saving'
                ? 'saving'
                : saveState === 'saved'
                  ? 'saved'
                  : 'unsaved'
            }
            state={saveState === 'unmodified' ? 'unsaved' : saveState}
```

Reply:
You are correct.